2-mandatory/6-writing-tests-advanced.js

```diff
@@ -48,13 +48,14 @@ function formatCourseworkResult(trainee) {
 - (Reminder: You must have run `npm install` one time before this will work!)
 */
 
-/*
-  Write a test that checks the output of formatCourseworkResult when passed the following trainee:
-  {
+
+//  Write a test that checks the output of formatCourseworkResult when passed the following trainee:
+test("trainee has a score of 63", () => {})  
+  const trainee = {
     name: "Xin",
     score: 63
-  }
-*/
+  };
```

Comment:
Think you are missing a bit with these tests....

Take a look at the expect function :)

Also-- I think there are about 4 more tests after this one which should be updated!

Reply:
I will look into this.